docs: add installation instructions for Tiptap UI Components (#228)
ueberdosis/tiptap-docs: src/content/ui-components/getting-started/style.mdx

---
title: Style Tiptap UI Components
meta:
  title: UI Components | Tiptap
  description: Style Tiptap UI Components to match your design system or application theme.
  category: UI Components
---

import { Callout } from '@/components/ui/Callout'

### Add Styles

> To enable support for SCSS, install the appropriate SCSS compiler based on your framework setup.

Reviewer: ```suggestion
Component styles are written in SCSS. To enable SCSS support, install the appropriate SCSS compiler for your framework.
```

<Callout title="Styles are only added when using the CLI" variant="info">
  Styles are included only if you installed a component or template using the CLI. These styles come

>   bundled with whatever you add through the CLI.{' '}

Reviewer: ```suggestion
  bundled with whatever you add through the CLI.
```

</Callout>

#### For Next.js

1. Install `sass`:

```bash
pnpm add -D sass
```

2. Import the SCSS partials in your main stylesheet:

> **File:** `global.css`

Reviewer: Before step 2, you have to create a `global.css` file and import it in the `layout.tsx` component, right?

Author: Its included when you create a new next.js project

```css
@import 'path-to/styles/_variables.scss';
@import 'path-to/styles/_keyframe-animations.scss';
```

#### For Vite + React

1. Install `sass-embedded`:

```bash
pnpm add -D sass-embedded
```

2. Import the SCSS partials in your main stylesheet:

**File:** `src/index.css`

```css
@import 'path-to/styles/_variables.scss';
@import 'path-to/styles/_keyframe-animations.scss';
```

Make sure the SCSS files are in the correct path so the styles load properly in your project.

> Let me know if you'd like to include support instructions for other tools or frameworks.

Reviewer: How can a customer let us know that they need an installation guide for another framework? If you want the customer to contact the Tiptap team, you have to provide it a way to do so.

I'd suggest telling customers to reach out to humans@tiptap.dev or, even better, to open an issue https://github.com/ueberdosis/tiptap-docs/issues 

@ThisDavidRichard am I right with this?
